demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanAssessmentStatementFragmentController.php

```diff
@@ -380,7 +380,7 @@ public function editStatementFragmentAjaxAction(
                 $updateData['mayChangeMetaData'] = false;
             }
             $updatedStatementFragment = $this->statementHandler->updateStatementFragment($fragmentId, $updateData, $isReviewer);
-            $returnCode = 200;
+            $returnCode = 400;
```

Comment:
My intention was to return a 400 status code if `getFragmentOfStatement` returns `null`, not always. Also please use the constants provided by `Symfony\Component\HttpFoundation\Response`.

Reply:
done in [74bd37a](https://github.com/demos-europe/demosplan-core/pull/2820/commits/74bd37a18126c403885926bcfd9aab97df4975a6)